### DISCUSSION TITLE
建议：多路推流（multi-rtmp）能否考虑在每条推流项目中设置不同场景或者封面？

### DISCUSSION THREAD

**Discussion by Thomism** [sorayuki/obs-multi-rtmp#246] [General]
关于多路推流（multi-rtmp），能否考虑在每条推流项目中选择不同场景或者封面？
最好是统一一个游戏进程，或者推流主体。然后，可以在不同的推流项目中选择有区别的封面。
因为有些推流平台要求他们自己的水印，不同平台的水印需要分开选择图片作为封面。

**Comment by ernanims** [comment]
Hello folks
My OBS version is 29.0.2 (64 bits) for Mac
I already tried to instal the obi-multi-rtmp, and nothing happen. It looks like that nothing has happened.
Is there any  help?
